tests/gnrc_rpl_srh/tests/01-run.py

```diff
@@ -372,7 +366,6 @@ def run(func):
     run(test_seq_left_0)
     run(test_time_exc)
     print("SUCCESS")
-    sniffer.stop()
```

Comment:
Why remove the `stop` call?

Reply:
Actually, `stop()` is not really necessary, so I removed it completely.